src/content/en/tools/lighthouse/audits/required-aria-attributes.md

```diff
@@ -0,0 +1,43 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Elements With ARIA Roles Have Required Attributes" Lighthouse audit.
+
+{# wf_updated_on: 2017-01-17 #}
+{# wf_published_on: 2017-01-17 #}
+
+# Elements With ARIA Roles Have Required Attributes  {: .page-title }
+
+## Why the audit is important {: #why }
+
+The `role` attribute helps screen readers understand the purpose of an ARIA
+widget. Each `role` has a set of required, additional attributes that
+describe the state of the widget to the screen reader.
+
+## How to pass the audit {: #how }
+
+<<_shared/query.md>>
+
+To find each element's missing required attributes:
+
+1. Note the `role` and `aria-*` attributes of the element.
+
+1. Go to [Definition of Roles][roles].
+
+1. Go to the page for this element's `role`.
+
+1. Check the `aria-*` attributes of the element against the **Required
+   States and Properties** list.
+
+1. Add any attributes that are missing.
+
+[qs]: /web/tools/chrome-devtools/console/command-line-reference#queryselector
+[qsa]: /web/tools/chrome-devtools/console/command-line-reference#queryselectorall
+[xp]: /web/tools/chrome-devtools/console/command-line-reference#xpath
+[roles]: https://www.w3.org/TR/wai-aria/roles#role_definitions
```

Comment:
yep https://www.w3.org/TR/wai-aria-1.1/#h-role_definitions. Also there are new roles in aria 1.1 that aren't in aria 1.0.